platform/ios/src/MGLMapView.mm

```diff
@@ -4045,6 +4052,9 @@ - (MGLMapCamera *)cameraForCameraOptions:(const mbgl::CameraOptions &)cameraOpti
     {
         options.pitch = camera.pitch;
     }
+    if (! MGLEdgeInsetsEqual(camera.padding, MGLEdgeInsetsZero)) {
```

Comment:
Not sure if this is correct, as there is edgePadding as argument as well as in padding in MGLMapCamera. The padding in MGLMapCamera overwrites the param here.